projects/ccf-rui/src/assets/docs/README.md

```diff
@@ -14,6 +14,9 @@ A first beta of the CCF RUI became available in October 2020. The CCF RUI specif
 
 The registration data is used in current versions of the Common Coordinate Framework (CCF, see [CCF Portal](https://hubmapconsortium.github.io/ccf/") for details) and the [CCF Exploration User Interface (EUI)](https://portal.hubmapconsortium.org/ccf-eui) developed within HuBMAP.
 
+# Standard Operating Procedure (SOP)
```

Comment:
This probably works with out a newline between the header and the text, but for consistency with the other sections, please add a newline.